WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginSiteCheckErrorFragment.kt

```diff
@@ -110,4 +120,16 @@ class LoginSiteCheckErrorFragment : Fragment(R.layout.fragment_login_site_check_
 
         unifiedLoginTracker.track(step = Step.NOT_A_JETPACK_SITE)
     }
+
+    private fun initBackPressHandler() {
```

Comment:
Warning (⚠️): This function is unused. Probably you wanted to use that after `initDagger()` just like it is being done for the `LoginNoSitesFragment` class.

Reply:
Great catch :) Thanks